data/json/furniture.json

```diff
@@ -283,7 +283,7 @@
     "type": "furniture",
     "id": "f_bulletin",
     "name": "bulletin board",
-    "description": "Pin some notes for other survivors to read.",
+    "description": "A big, cork bulletin board sporting various notices, most of them depressing.  Pin some notes for other survivors to read.",
```

Comment:
The primary use of a bulletin board is as a camp message board, so you probably don't want such a specific description.